cirq/contrib/paulistring/convert_gate_set.py

```diff
@@ -27,11 +27,11 @@ def converted_gate_set(circuit: circuits.Circuit,
     CZ/PauliInteractionGate, PauliStringPhasor}.
     """
     conv_circuit = circuits.Circuit(circuit)
-    optimizers.ConvertToCzAndSingleGates().optimize_circuit(conv_circuit)
-    optimizers.MergeSingleQubitGates().optimize_circuit(conv_circuit)
+    optimizers.ConvertToCzAndSingleGates()(conv_circuit)
```

Comment:
Keep the `optimize_circuit`; it's clearer. This applies everywhere in the PR. Ssearch for `()(`